src/style.css

```
@@ -1 +1,49 @@
-/* add styles here */
+html {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.stars__star {
+  box-sizing: border-box;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  padding-right: 4px;
+}
+
+.stars__star:last-child {
+  padding-right: 0;
+}
+
+.stars--1 li:nth-child(-n+1) {
```

```suggestion
.stars--1 li:nth-child(-n+1),
...,
stars--5 li:nth-child(-n+5) {
```
Consider using commas to group your selectors